assets/js/blocks/cart-checkout/cart/full-cart/cart-line-item-row.js

```diff
@@ -26,7 +26,13 @@ const calcPriceDifference = ( subtotal, total ) => {
 
 const ProductVariationDetails = ( { variation } ) => {
 	const variationsText = variation
-		.map( ( v ) => `${ v.attribute }: ${ v.value }` )
+		.map( ( v ) => {
+			if ( v.attribute ) {
+				return `${ v.attribute }: ${ decodeEntities( v.value ) }`;
```

Comment:
The attribute name also needs to be decoded:
```suggestion
				return `${ decodeEntities( v.attribute ) }: ${ decodeEntities( v.value ) }`;
```